### PR TITLE
skipping winograd in single gpu test

### PR DESCRIPTION
**Pull request by zstreet87**
Workaround: https://ontrack-internal.amd.com/browse/SWDEV-480036

### CODE REVIEW

**Comment by jayfurmanek** [comment]
Can you also pull in the changes that we have in the other branches:
https://github.com/ROCm/tensorflow-upstream/commit/d16b9793d021ea138709d7b99c60e07582e74575

**Comment by zstreet87** [comment]
@jayfurmanek done

**Comment by jayfurmanek** [comment]
This maybe needs a rebase now

**Comment by zstreet87** [comment]
@jayfurmanek done

File: tensorflow/tools/ci_build/linux/rocm/run_gpu_single.sh [hunk 1]
```diff
@@ -51,7 +51,22 @@ export TF_PYTHON_VERSION=$PYTHON_VERSION
 export TF_NEED_ROCM=1
 export ROCM_PATH=$ROCM_INSTALL_DIR
 
-yes "" | $PYTHON_BIN_PATH configure.py
```
Comment (jayfurmanek) [inline tensorflow/tools/ci_build/linux/rocm/run_gpu_single.sh:54]: the configure should end up in the else path

File: tensorflow/tools/ci_build/linux/rocm/run_gpu_single.sh [hunk 2]
```diff
@@ -37,7 +37,7 @@ if [[ -n $1 ]]; then
     ROCM_INSTALL_DIR=$1
 else
     if [[ -z "${ROCM_PATH}" ]]; then
-        ROCM_INSTALL_DIR=/opt/rocm-6.2.0
```
Comment (jayfurmanek) [inline tensorflow/tools/ci_build/linux/rocm/run_gpu_single.sh]: lets keep 6.2.0 as default

File: tensorflow/tools/ci_build/linux/rocm/run_gpu_single.sh (hunk 1 above)
Comment (zstreet87) [reply tensorflow/tools/ci_build/linux/rocm/run_gpu_single.sh:54]: I don't understand what you mean. Did I nuke the wrong commit when managing the conflict?

Comment (jayfurmanek) [reply tensorflow/tools/ci_build/linux/rocm/run_gpu_single.sh:54]: might be just a merge conflict/artifact. Either way the else ("legacy") path needs to call configure.

Comment (zstreet87) [reply tensorflow/tools/ci_build/linux/rocm/run_gpu_single.sh:54]: I just added what is in that commit you wanted. The `configure.py` logic was already under the else path so still not sure what you mean - but I didn't have the rest of the else logic from that commit (deleted wrong stuff) so added it.

Comment (jayfurmanek) [reply tensorflow/tools/ci_build/linux/rocm/run_gpu_single.sh:54]: nevermind - I read the diff wrong. This is good

**Review by jayfurmanek — APPROVED** [review]

**Pull request merged**